chevah/compat
# [#678] Add absolute path API.

Scope
=====

Fixes #678 
Fixes #677 
Fixes #115 

We need a custom API to detect if a path is absolute.

And as part of this PR, we might need to add support for handling windows shares as segments.


Changes
=======

As a drive-by I have updated brink.sh and conf.
Also updated the GitHub actions workflow with latest oses.



How to try and test the changes
===============================

reviewers: @danuker 

check that changes make sense.
